admin.php

```diff
@@ -354,7 +354,8 @@
             . html_writer::link($slink, get_string("page_link_backtoconfig", "block_coursefeedback")) . "</div>");
         $active = get_config("block_coursefeedback", "active_feedback");
         $table = new html_table();
-        $table->head = array("ID",
+        $table->head = array(
```

Comment:
Nichts falsch hier. Das ist für mich nur ein Aufhänger für das allgemeine Thema der [Array-Notation](https://moodledev.io/general/development/policies/codingstyle#array-syntax). Da soweit ich das sehen konnte im ganzen Plugin durchgehend die alte Schreibweise (`array(...)`) verwendet wird, ist das sowieso in Ordnung, das hier auch zu machen. Aber falls wir uns irgendwann an die Style Guidelines von Moodle halten wollen bzw. sogar irgendwann deren CI Pipeline verwenden wollen, sollte auf die Kurzschreibweise umgestellt werden (`[...]`). Ist ja auch viel besser lesbar. ;) Aber das nur als Anmerkung am Rande.

Reply:
Fände ich gut, wenn wir das umstellen und würde vorschlagen, dass später ein Commit fertig gemacht wird, der alle Arrays in die richtige Notation bringt.

Reply:
Ja das sehe ich auch so und hab es tatsächlich auch auf schon auf meiner TODO Liste. Ich würde das dann alles in einem commit machen und an dieser Stelle für den Moment so lassen.